src/Http/Controllers/StaticPagesController.php

```diff
@@ -2,33 +2,123 @@
 
```

Comment:
src/Http/Controllers/StaticPagesController.php
Put admin controllers to subnamespace. Like this:
`src/Http/Controllers/Admin/StaticPagesController.php`